src/views/start/index.jsx

```diff
@@ -151,14 +174,18 @@ export default class Start extends Component {
             type="text"
             name="queryParameter"
             id="queryParameter"
-            defaultValue="bugfix"
+            defaultValue={this.queryValue}
+            onKeyDown={this.onQueryKeyDown.bind(this)}
```

Comment:
this could be just "this.onQueryKeyDown"

Reply:
can you try this on your machine? on mine it doesn't work anymoreif i remove '.bind(this)'. it's the same for the onFileExtensionKeyDown